app/templates/components/youtube-videos.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="row">
+  <div class="col-xs col-lg-9">
+    <YoutubeVideo @youtubeId="{{mainVideo.videoId}}" @title="{{mainVideo.title}}" />
+  </div>
+  <div class="col-xs col-lg-3">
+    <div class="about-alpha">
+      <div class="about-alpha__container">
+        {{#each sideVideos as |video|}}
+          <a class="about-alpha__video" href="https://youtu.be/{{video.videoId}}" target="_blank" title="{{video.title}}" {{action "changeVideo" video}}>
```

Comment:
I don't think it needs to be a link anymore since we do not link to youtube anymore